packages/ui/src/components/TextAreaEditor.svelte

```diff
@@ -57,16 +58,19 @@
       {placeholder}
       {height}
       {width}
+      {disabled}
       bind:this={inputRef}
       bind:value
       on:keydown={onKeydown}
       noFocusBorder={true}
     />
   </div>
-  <div class="flex-row-center mt-3">
-    <Button label={submitLabel} kind="no-border" size="medium" on:click={submit} />
-    <div class="ml-2" on:click={cancel}>
-      <ActionIcon icon={IconClose} size="medium" action={cancel} />
+  {#if !disabled}
```

Comment:
What if value will be passed, we should see it on screen even for disabled.

Reply:
Yes, the value will be displayed. I only hide Save/Cancel buttons when input is disabled.